README.md

```diff
@@ -4,6 +4,64 @@ or better yet, "can-in-the-middle" is system using the TU Truck Cape, a BeagleBo
 ## Setting up the Hardware
 The hardware for this project is 100% open. You can purchase every part of the hardware and hand assemble the pieces. The schematic for the Truck Cape or the CAN Man-in-the-middle is shown in the docs folder. There are multiple hardware versions for different form factors and switching functions. All versions use the BeagleBone Black and have 2 CAN channels and 2 J1708 channels. For more details, see the ![docs](docs) folder.
 
+### Controlling Relays
```

Comment:
neat.

It looks like there are combinations of gpio output settings that would be 'illegal' or, at least, wouldn't accomplish much: e.g. connecting CAN2 B-side to can0 and also bridging it to A side (both `echo 1 > /sys/class/gpio/gpio70/value` and `echo 1 > /sys/class/gpio/gpio76/value`)

I suggest you write a shell script or python script to capture the mapping of 'useful mode' to GPIO settings. The user will be less interested in the function of each switch and more interested in how to set the modes of operation of the cape.

Reply:
I agree. I'll do that in a new branch.